src/wmtk/operations/polygon_mesh/Splice.cpp

```diff
@@ -0,0 +1,156 @@
+#include "Splice.hpp"
+
+
+namespace {
+// Implicit opposite map for halfedges paired as [e] = {2*e, 2*e + 1}
+long implicit_opp(long h)
+{
+    return ((h % 2) == 0) ? (h + 1) : (h - 1);
+}
+} // namespace
+
+namespace wmtk::operations::polygon_mesh {
+
+Splice::Splice(Mesh& m, const Tuple& h, const Tuple& g, const OperationSettings<Splice>& settings)
+    : AtomicOperation(m)
+    , m_first_tuple(h)
+    , m_second_tuple(g)
+    , m_settings(settings)
+{}
+
+std::string Splice::name() const
+{
+    return "polygon_mesh_splice";
+}
+
+bool Splice::execute()
+{
+    assert(precondition());
+
+    // Old halfedge indices
+    long old_h_id = get_halfedge_from_tuple(m_first_tuple);
+    long old_g_id = get_halfedge_from_tuple(m_second_tuple);
+    long old_hn_id = get_next(old_h_id);
+    long old_gn_id = get_next(old_g_id);
+
+    // Do nothing for identical halfedges
+    if (old_hn_id == old_gn_id) {
```

Comment:
in general (i think you did this in other places), returning true means the operation succeeded and something happened. In this case I think you want to return `false` because the operation didn't happen.

Reply:
The operation did happen; it's just trivial in this case as the splice is just swapping something with itself. I would prefer to return true for this operation since the operation technically succeeded.